src/main/java/org/isf/admission/manager/AdmissionBrowserManager.java

```diff
@@ -508,4 +507,8 @@ else if (dateOut.isBefore(ad.getAdmDate()) || dateOut.equals(ad.getAdmDate())) {
 			throw new OHDataValidationException(errors);
 		}
 	}
+	public boolean isAdmissionActiveOnDate(Admission admission, LocalDateTime checkDate) {
```

Comment:
I see little value in extracting the conditional into a method given the overhead that is required to invoke the method.  I don't believe this should be done.